61/ShubhamAshish/URLShortner.py

```diff
@@ -0,0 +1,15 @@
+import streamlit as st
+import pyperclip
+import pyshorteners
+
+
+
+st.title("URL Shortner")
+st.write("Enter the URL to shorten")
+url = st.text_input("URL")
+if st.button("Shorten"):
+    shortener = pyshorteners.Shortener()
```

Comment:
I wrote one from scratch the other day, it makes sense to use a library, nice. I would put any code in functions for easier re-use.